lib/listen.js

```diff
@@ -61,6 +61,14 @@ function listen(appDirectory, sessionStore, preSessionInject, postSessionInject,
     let appStartList = amorphicOptions.appStartList;
     let promises = [];
 
+    for (let app in appStartList) {
+        if (!appList[app]) {
+            console.error('FATAL your main application is not in the applications list in your root config.');
+            console.error('Amorphic is shutting down.');
+            process.exit(1);
```

Comment:
I think the console.error is sufficient - otherwise i'm envisioning containers infinitely looping in CI when someone realizes they forgot this setting

Reply:
ah good point will remove the `process.exit(1);`